examples/projections/cyl/cyl_oblique_mercator_1.py

```diff
@@ -0,0 +1,27 @@
+"""
+Oblique Mercator
+================
+
+Oblique configurations of the cylinder give rise to the oblique Mercator
+projection. It is particularly useful when mapping regions of large lateral
+extent in an oblique direction. Both parallels and meridians are complex
+curves. The projection was developed in the early 1900s by several workers.
+
+**oa**\ *lon0/lat0* */azimuth*\ */scale*\ **[+v]** or
+**OA**\ *lon0/lat0* */azimuth*\ */width*\ **[+v]** or
```

Comment:
```suggestion
**oa**\ |**oA**\ *lon0/lat0* */azimuth*\ */scale*\ **[+v]** or
**Oa**\ |**OA**\ *lon0/lat0* */azimuth*\ */width*\ **[+v]** or
```

Capitalizing `o` and `a` are independent of one another, but the way it currently looks makes it seem like they are changed together.